alerts/alerts.libsonnet

```diff
@@ -24,6 +24,43 @@
           },
         ],
       },
+      {
+        name: 'mintel-disks',
+        rules: [
+          {
+            alert: 'KubePersistentVolumeInodeUsageCritical',
+            expr: |||
+              mintel:pvc:inodes_free:percentage <= 5
+            |||,
+            'for': '5m',
+            labels: {
+              severity: 'warning',
```

Comment:
should probably be critical

Reply:
also don't need the context label, it was a legacy thing

Reply:
Done